Essentials/src/com/earth2me/essentials/Console.java

```diff
@@ -8,7 +8,7 @@
 
 
 public final class Console implements IMessageRecipient {
-    public static final String NAME = "Console";
+    public static  String NAME = "Console";
```

Comment:
Get rid of this. The display name should be obtained from the `ess` instance when it is called from `getDisplayName()` for example.

Another thing to consider is that this probably should be localized.